engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -38,7 +38,6 @@
     private Set<StateChange> desiredStateChanges = Sets.newLinkedHashSet();
     private Set<StateChange> desiredStateResets = Sets.newLinkedHashSet();
     private Map<ResourceUrn, BaseFBOsManager> fboUsages = Maps.newHashMap();
-    private NodeTask task;
```

Comment:
Once the task classes have been removed, in the following commit you can remove/replace all lines that used that functionality.